examples/get_event.json

```diff
@@ -104,24 +117,45 @@
             "result": true,
             "name": "DediPath"
           }
+        },
+        "v6": {
+          "address": "2001:db8:3333:4444:5555:6666:7777:8888",
+          "geolocation": {
+            "accuracyRadius": 5,
+            "latitude": 49.982,
+            "longitude": 36.2566,
+            "postalCode": "10112",
+            "timezone": "Europe/Berlin",
+            "city": {
+              "name": "Berlin"
+            },
+            "country": {
+              "code": "DE",
+              "name": "Germany"
+            },
+            "continent": {
+              "code": "EU",
+              "name": "Europe"
+            },
+            "subdivisions": [
+              {
+                "isoCode": "BE",
+                "name": "Land Berlin"
+              }
+            ]
+          },
+          "asn": {
+            "asn": "6805",
+            "name": "Telefonica Germany",
+            "network": "2a02:3100::/24"
+          },
+          "datacenter": {
+            "result": false,
+            "name": ""
+          }
         }
       }
     },
-    "incognito": {
```

Comment:
In the spec I still see incognito as required, but it's removed here in the example.

Reply:
Nevermind, saw it moved further down. All the required fields appear to be here 👍